.github/workflows/ci.yml

```
@@ -110,3 +112,38 @@ jobs:
         timeout_minutes: 15
         max_attempts: 2
         command: ./tools/test_monitor/run-tests.sh
+  flow-legacy-client-compatibility-test:
+    name: Client Compatibility Tests With Observer
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version:
+          - 1.16
+          - 1.17
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version}}
```

The go version management has changed recently. This should use the same versions as the other suites
```suggestion
    strategy:
      fail-fast: false
    runs-on: ubuntu-latest
    steps:
    - name: Setup Go
      uses: actions/setup-go@v2
      with:
        go-version: ${{ env.GO_VERSION }}
```

---

Thank you for the update. I integrated the change into our items.

Reference: https://github.com/GetElastech/flow-go/commit/9303250d5d652a0244c67e988aa810b7e6975fd2